### PR TITLE
CHANGELOG for 0.0.6

### PR DESCRIPTION
**Pull request by xorye**
Signed-off-by: David Kwon <dakwon@redhat.com>

### CODE REVIEW

File: microprofile.ls/CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,50 @@
...
+ * Fixed duplicated comments on range formatting. See [#233](https://github.com/redhat-developer/quarkus-ls/pull/233)
+ * Empty completion after an error from microprofile/projectInfo. See [#228](https://github.com/redhat-developer/quarkus-ls/issues/228)
+
+### Build
+
+ * com.redhat.microprofile.ls cannot be released to JBoss Nexus. See [#226](https://github.com/redhat-developer/quarkus-ls/issues/226)
+
+### Others
+
+ * Deploy test projects to download.jboss.org. See [#289](https://github.com/redhat-developer/quarkus-ls/pull/289)
```
Comment (fbricon) [inline microprofile.ls/CHANGELOG.md:36]: should be in build

File: microprofile.ls/CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,50 @@
...
+ * Remove Quarkus name in MicroProfile LS. See [#262](https://github.com/redhat-developer/quarkus-ls/issues/262)
+ * Refactor unit test so that they can be reused. See [#258](https://github.com/redhat-developer/quarkus-ls/pull/258)
```
Comment (fbricon) [inline microprofile.ls/CHANGELOG.md:38]: should be in build

File: microprofile.ls/CHANGELOG.md [hunk 3]
```diff
@@ -1,6 +1,50 @@
...
+ * Support for `MicroProfile OpenTracing` properties. See [#240](https://github.com/redhat-developer/quarkus-ls/issues/240)
```
Comment (fbricon) [inline microprofile.ls/CHANGELOG.md:39]: enhancement

File: microprofile.ls/CHANGELOG.md [hunk 4]
```diff
@@ -1,6 +1,50 @@
...
+ * Consume LSP4J 0.9.0. See [#237](https://github.com/redhat-developer/quarkus-ls/issues/237)
```
Comment (fbricon) [inline microprofile.ls/CHANGELOG.md:40]: build

File: microprofile.ls/CHANGELOG.md [hunk 5]
```diff
@@ -1,6 +1,50 @@
...
+ * Provide codeLens participant. See [#232](https://github.com/redhat-developer/quarkus-ls/pull/232)
```
Comment (fbricon) [inline microprofile.ls/CHANGELOG.md:41]: enhancement

File: microprofile.ls/CHANGELOG.md [hunk 6]
```diff
@@ -1,6 +1,50 @@
...
+ * Provide hover participant. See [#231](https://github.com/redhat-developer/quarkus-ls/pull/231)
```
Comment (fbricon) [inline microprofile.ls/CHANGELOG.md:42]: enhancement

File: microprofile.ls/CHANGELOG.md [hunk 7]
```diff
@@ -1,6 +1,50 @@
...
+ * Split Quarkus / MicroProfile support. See [#229](https://github.com/redhat-developer/quarkus-ls/issues/229)
```
Comment (fbricon) [inline microprofile.ls/CHANGELOG.md:43]: build

File: microprofile.ls/CHANGELOG.md [hunk 8]
```diff
@@ -1,6 +1,50 @@
...
+ * Upversion to 0.0.6-SNAPSHOT. See [#218](https://github.com/redhat-developer/quarkus-ls/pull/218)
```
Comment (fbricon) [inline microprofile.ls/CHANGELOG.md:44]: not in changelog

File: microprofile.ls/CHANGELOG.md [hunk 9]
```diff
@@ -1,6 +1,50 @@
...
+ * Support for MicroProfile RestClient/Health Diagnostics. See [#217](https://github.com/redhat-developer/quarkus-ls/issues/217)
```
Comment (fbricon) [inline microprofile.ls/CHANGELOG.md:45]: enhancement

File: microprofile.ls/CHANGELOG.md [hunk 10]
```diff
@@ -1,6 +1,50 @@
...
+ * Empty completion after an error from microprofile/projectInfo. See [#228](https://github.com/redhat-developer/quarkus-ls/issues/228)
+
+### Build
+
+ * com.redhat.microprofile.ls cannot be released to JBoss Nexus. See [#226](https://github.com/redhat-developer/quarkus-ls/issues/226)
+
+### Others
+
+ * Deploy test projects to download.jboss.org. See [#289](https://github.com/redhat-developer/quarkus-ls/pull/289)
+ * Remove Quarkus name in MicroProfile LS. See [#262](https://github.com/redhat-developer/quarkus-ls/issues/262)
```
Comment (fbricon) [inline microprofile.ls/CHANGELOG.md:37]: build

File: microprofile.ls/CHANGELOG.md [hunk 11]
```diff
@@ -1,6 +1,50 @@
...
+ * Support for `MicroProfile Metrics` properties . See [#241](https://github.com/redhat-developer/quarkus-ls/issues/241)
+ * CodeAction to Generate Open API Annotations. See [#239](https://github.com/redhat-developer/quarkus-ls/issues/239)
+ * Support for `MicroProfile Health` CodeAction. See [#236](https://github.com/redhat-developer/quarkus-ls/issues/236)
+ * Support for `MicroProfile Open API` properties. See [#216](https://github.com/redhat-developer/quarkus-ls/issues/216)
+ * Improve performance while getting dependencies of deployment JARs. See [#207](https://github.com/redhat-developer/quarkus-ls/issues/207)
+
+### Bug Fixes
+
+ * Duplicate static properties after saving Java files. See [#301](https://github.com/redhat-developer/quarkus-ls/issues/301)
+ * Allow exclude unknown property pattern * to match forward slashes. See [#284](https://github.com/redhat-developer/quarkus-ls/pull/284)
```
Comment (fbricon) [inline microprofile.ls/CHANGELOG.md:21]: excluded or exclusion

File: microprofile.ls/CHANGELOG.md [hunk 12]
```diff
@@ -1,6 +1,50 @@
...
+ * Hide OpenAPI source action if it is not applicable. See [#280](https://github.com/redhat-developer/quarkus-ls/issues/280)
+ * Filter for Java (server) snippet . See [#265](https://github.com/redhat-developer/quarkus-ls/issues/265)
```
Comment (fbricon) [inline microprofile.ls/CHANGELOG.md:23]: enhancement?

File: microprofile.ls/CHANGELOG.md [hunk 13]
```diff
@@ -1,6 +1,50 @@
...
+ * Missing exports for reusing unit tests. See [#263](https://github.com/redhat-developer/quarkus-ls/issues/263)
```
Comment (fbricon) [inline microprofile.ls/CHANGELOG.md:24]: build

File: microprofile.ls/CHANGELOG.md [hunk 14]
```diff
@@ -1,6 +1,50 @@
...
+ * Support for `java.math.BigDecimal` values. See [#261](https://github.com/redhat-developer/quarkus-ls/issues/261)
```
Comment (fbricon) [inline microprofile.ls/CHANGELOG.md:25]: enhancement

File: microprofile.ls/CHANGELOG.md (hunk 8 above)
Comment (fbricon) [reply microprofile.ls/CHANGELOG.md:44]: still need to be removed from changelog

**Comment by xorye** [comment]
Fixed

**Comment by fbricon** [comment]
https://github.com/redhat-developer/quarkus-ls/pull/235 and https://github.com/redhat-developer/quarkus-ls/pull/290 were missing from the changelog, I've added them, so you can get them in the changelog

**Comment by xorye** [comment]
updated

File: microprofile.ls/CHANGELOG.md (hunk 8 above)
Comment (fbricon) [reply microprofile.ls/CHANGELOG.md:46]: still need to be removed from changelog

Comment (xorye) [reply microprofile.ls/CHANGELOG.md]: sorry, fixed

**Comment by fbricon** [comment]
thanks @xorye !

**Pull request merged**